arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,226 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
+
+## Setting up your development environment
+
+### Prerequisites
 
 #### Rust toolchain
 
-Follow the instructions on [Rust Lang’s installation page](https://www.rust-lang.org/tools/install) to get a full Rust toolchain installed on your system. Make sure after installation that you have access to the programs `rustup`, `rustc`, and `cargo` from your preferred command line terminal (programs should be added to your system’s PATH, more instructions available on Rust’s website)
+Follow the instructions on [Rust Lang’s installation page](https://www.rust-lang.org/tools/install) to install a complete Rust toolchain on your system. After installation, ensure you have access to the programs `rustup`, `rustc`, and `cargo` from your preferred terminal application.
 
 #### VS Code
 
-We recommend VSCode as the IDE of choice for developing Stylus contracts for its excellent Rust support. See **[code.visualstudio.com](https://code.visualstudio.com/)** to install. Feel free to use another text editor or IDE if you’re comfortable with those.
+We recommend [VSCode](https://code.visualstudio.com/) as the IDE of choice for its excellent Rust support, but feel free to use another text editor or IDE if you’re comfortable with those.
 
 Some helpful VS Code extensions for Rust development:
 
-- [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)
-- [Error Lens](https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens)
-- [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml)
-- [Dependi](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi)
-  
-#### Testnet ETH for deployment
-
-You’ll need some testnet ETH for deploying your Rust contract for live testing. Explained below in further detail.
+- [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer): Provides advanced features like smart code completion and on-the-fly error checks
+- [Error Lens](https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens): Immediately highlights errors and warnings in your code
+- [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml): Improves syntax highlighting and other features for TOML files, often used in Rust projects
+- [Dependi](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi): Helps manage Rust crate versions directly from the editor
 
-#### Developer wallet / account
+#### Docker
 
-When deploying on and interacting with a testnet chain, it’s important to use a fresh wallet that does not contain any real assets. You’ll often be including private keys as CLI arguments to execute transactions programmatically, **_so avoid using personal accounts for development_**.
+`cargo stylus` requires Docker to operate. You can download Docker from [Docker’s website](https://www.docker.com/products/docker-desktop).
```

Comment:
I would maybe say that "some `cargo stylus` commands require Docker to operate", since it's not a requirement of cargo-stylus per se, but just using some commands for reproducible builds.

Reply:
Noted, fixed.